src/test/scala/sigmastate/utxo/ErgoLikeInterpreterSpecification.scala

```diff
@@ -31,8 +31,8 @@ class ErgoLikeInterpreterSpecification extends SigmaTestingCommons {
 
     val ctx = ErgoLikeContext.dummy(fakeSelf)
 
-    val e = compile(Map("h1" -> h1.bytes, "h2" -> h2.bytes), "h1 == h1")
-    val exp = EQ(ByteArrayConstant(h1.bytes), ByteArrayConstant(h1.bytes))
+    val e = compileWithCosting(Map("h1" -> h1.bytes, "h2" -> h2.bytes), "h1 == h1")
+    val exp = TrueLeaf
```

Comment:
@greenhat I think this is the result of constant propagation in Special.
It can be [turned off](https://github.com/ScorexFoundation/sigmastate-interpreter/blob/7ba836ae5ddb9d915cfa4b9e68d83fe93bcdda0a/src/main/scala/sigmastate/eval/IRContext.scala#L15)

Reply:
@ergomorphic Thanks for the hint.  Nope, enabling constant propagation does not change this case, moreover, it breaks `1 + 1` case here http://github.com/ScorexFoundation/sigmastate-interpreter/blob/b9ccb7e8d05e95e5c3183f46dc9b81790f0b685b/src/test/scala/sigmastate/eval/ErgoTreeBuildingTest.scala#L28 (it produces `IntConstant(2)` instead of `Plus(...)`)